class.jetpack.php

```diff
@@ -4032,6 +4032,24 @@ function admin_page_load() {
 					wp_safe_redirect( Jetpack::admin_url( array( 'page' => $redirect ) ) );
 				}
 				exit;
+			case 'onboard' :
+				$user_can = is_multisite()
```

Comment:
Maybe call the var `$user_can_manage_options` (or just `$can_manage_options`)?

Reply:
Removed the var completely as part of the simplification in https://github.com/Automattic/jetpack/pull/8352/commits/ae2284fb9d1f6ddf003e805409be7dd615d8f4a2.